web_external/Datasets/CreateDatasetView.js

```diff
@@ -179,8 +160,8 @@ const CreateDatasetView = View.extend({
         }
     },
 
-    uploadFinished: function (files) {
-        this.uploadedZipFiles = _.pluck(files.files, 'name');
+    uploadFinished: function (info) {
+        this.uploadedFiles = info.files;
```

Comment:
Sounds good. It's probably not a good idea to trust that strange system-created objects will be immutable.